test/OpenTelemetry.Resources.Container.Tests/OpenTelemetry.Resources.Container.Tests.csproj

```diff
@@ -7,12 +7,14 @@
     <Nullabe>enable</Nullabe>
   </PropertyGroup>
 
-  <ItemGroup>
-    <ProjectReference Include="$(RepoRoot)\src\OpenTelemetry.Resources.Container\OpenTelemetry.Resources.Container.csproj" />
+  <ItemGroup Condition="'$(TargetFramework)' != '$(NetFrameworkMinimumSupportedVersion)'">
```

Comment:
Is this condition really needed? You are targeting only .NET 6+ here.

Reply:
Not needed. Removed.